app/views/layouts/_sidebar.html.erb

```diff
@@ -9,6 +9,9 @@
       <div class="list-group list-group-flush">
         <a href="0" class="list-group-item active">Active Link Example</a>
         <a href="0" class="list-group-item">Dummy Link</a>
+        <% if current_user&.casa_admin? %>
+          <%= link_to "Admins", casa_admins_path, class: "list-group-item" %>
+        <% end %>
```

Comment:
I believe `link_to_if` would work here as well:

```erb
<%= link_to_if current_user&.casa_admin?, "Admins", casa_admins_path, class: "list-group-item" %>
```

Reply:
I like this much better! Thanks Diego! I had to pass an empty block to it so nothing would render if `current_user&.casa_admin?` is false. Without the empty block, the plain text "Admins" was still rendering if `current_user&.casa_admin?` is false.

Reply:
you're right, my initial understanding of `link_to_if` was incorrect 🤦🏻‍♂️